source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,371 @@
+- **Azure Linux**: `azl3/`
+- **OpenSUSE**: `opensuse15/` (CUDA 13.0+ only)
+
+### Key Differences by Distribution Type
+
+**Ubuntu/Debian distributions:**
+
+- Use `apt-get install` commands
+- Repository setup uses GPG keys and `.list` files
+
+**RHEL/CentOS/Rocky Linux distributions:**
+
+- Use `yum install` or `dnf install` commands
+- Repository setup uses `.repo` configuration files
+- Include repository files: `cuda.repo-x86_64`, `cuda.repo-arm64`
+
+### Installing CUDA components on your container
+
+1. Navigate to `dist/{cuda_version}/{your_os}/base/` or `runtime/` in the repository
+2. Open the `Dockerfile` for your target distribution
+3. Copy all `ENV` variables for package versions and NVIDIA Container Toolkit support
+4. Copy the `RUN` commands for installing the packages.
+5. If you are using the `runtime` components, make sure to copy the `ENV` and `RUN` commands from the `base` Dockerfile as well.
+6. For RHEL-based systems, also copy any `.repo` configuration files needed.
+
+```{note}
+Package versions change between CUDA releases. Always check the specific Dockerfile for your desired CUDA version and distribution to get the correct versions.
+```
+
+### Installing RAPIDS libraries on your container
+
+Refer to the Docker Templates in the [Custom RAPIDS Docker Guide](../custom-docker.md) to configure your RAPIDS installation, adding the conda or pip installation commands after the CUDA components are installed.
+
+## Essential Environment Variables
+
+These environment variables are **required** for CUDA functionality on the container and access to the host system's GPUs through the NVIDIA Container Toolkit
+
+| Variable                     | Purpose                          |
+| ---------------------------- | -------------------------------- |
+| `NVIDIA_VISIBLE_DEVICES`     | Specifies which GPUs are visible |
+| `NVIDIA_DRIVER_CAPABILITIES` | Required driver capabilities     |
+| `NVIDIA_REQUIRE_CUDA`        | Driver version constraints       |
+| `PATH`                       | Include CUDA binaries            |
+| `LD_LIBRARY_PATH`            | Include CUDA libraries           |
+
+## Complete Integration Examples
+
+Here are complete examples showing how to build a RAPIDS container with CUDA 12.9.1 components on an Ubuntu 24.04 base image for both conda and pip installations:
+
+### RAPIDS with conda (Base Components)
+
+Create an `env.yaml` file alongside your Dockerfile with your desired RAPIDS packages following the configuration described in the [Custom RAPIDS Docker Guide](../custom-docker.md). Set the `TARGETARCH` build argument to match your target architecture (`amd64` for x86_64 or `arm64` for ARM processors).
+
+```dockerfile
+FROM ubuntu:24.04
+
+# Build arguments
+ARG TARGETARCH=amd64
+
+# Architecture detection and setup
+ENV NVARCH=${TARGETARCH/amd64/x86_64}
+ENV NVARCH=${NVARCH/arm64/sbsa}
+
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
+LABEL maintainer="NVIDIA CORPORATION <cudatools@nvidia.com>"
```

Comment:
Do we need this here? I'm not too familiar on how LABEL work, but I wonder if this could confuse a user